### ISSUE TITLE
Faster test runs by suppressing output.

### ISSUE THREAD

**Issue by ioquatix** [ioquatix/sus#31]
One of the slowest parts of running tests is collecting output.

If a test is successful, the output would never even be looked at.

I believe we can improve performance significantly by doing the following:

- Running all tests with null output.
- Collect failed tests.
- Re-run them with verbose logging and output buffering.
- Use that to log failures.

The question is whether we can make running the test suite with null output significantly faster.

We should also have hooks for verbose logging, e.g. enabling `Console` logger to be verbose. We might need to use a standard hook, e.g. `$VERBOSE` and/or `$DEBUG` should be set.

**Comment by emiltin** [comment]
We use sus for testing external equipment. Rerunning test would be confusing as there is no guarantee external systems responds exactly the same each time.

**Comment by ioquatix** [comment]
Yeah, I experimented with this and the performance win didn't see worth it.
